lms/static/js/learner_dashboard/collections/certificate_collection.js

```diff
@@ -0,0 +1,12 @@
+(function (define) {
+  'use strict';
+  define([
+          'backbone',
+          'js/learner_dashboard/models/certificate_model'
+    ],
+    function (Backbone, Certficate) {
+      return Backbone.Collection.extend({
+          model: Certficate
+      });
+  });
+}).call(this, define || RequireJS.define);
```

Comment:
Nitpick: please ensure 4 spaces are used for indentations, I see there are some 2 space indent here